libraries/core_libs/network/src/tarcap/packets_handlers/pbft_sync_packet_handler.cpp

```diff
@@ -25,20 +25,17 @@ PbftSyncPacketHandler::PbftSyncPacketHandler(std::shared_ptr<PeersState> peers_s
       delayed_sync_events_tp_(1, true) {}
 
 void PbftSyncPacketHandler::validatePacketRlpFormat(const PacketData &packet_data) const {
-  // TODO[1561]: the way we handle PbftSyncPacket must be changed as we are using empty rlp as valid data -> it means
-  // pbft
-  //             syncing is complete. This is bad - someone could simply send empty PbftSyncPacket over and over again
-  if (packet_data.rlp_.itemCount() == 0) {
+  if (packet_data.rlp_.itemCount() == 1) {
```

Comment:
This requires comment. After few weeks/months no one will understand why we return here in case itemCount == 1

Reply:
No itemCount == 1 anymore, removed